migrations/outputs_wavefront/migration.go

```diff
@@ -0,0 +1,76 @@
+package outputs_wavefront
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/influxdata/toml"
+	"github.com/influxdata/toml/ast"
+
+	"github.com/influxdata/telegraf/migrations"
+)
+
+// Migration function
+func migrate(tbl *ast.Table) ([]byte, string, error) {
+	// Decode the old data structure
+	var plugin map[string]interface{}
+	if err := toml.UnmarshalTable(tbl, &plugin); err != nil {
+		return nil, "", err
+	}
+
+	// Check for deprecated option(s) and migrate them
+	var applied bool
+	msg := ""
+
+	rawHost, foundHost := plugin["host"]
+	rawPort, foundPort := plugin["port"]
+
+	if foundHost && foundPort {
```

Comment:
Hmmm shouldn't we also migrate if only a host is specified? We don't necessarily need a port, do we?